DESCRIPTION

```diff
@@ -18,9 +18,12 @@ Imports:
 RoxygenNote: 6.1.1
 Roxygen: list(markdown = TRUE)
 Suggests:
+    dplyr,
+    ggplot2,
```

Comment:
Hum, if we have to add them to suggests, maybe I should try making a version with only base R. I know we're supposed to add packages that are required to the vignette but I thought we could get away with it here...